apps/web/src/components/Composer/TextEditor/index.tsx

```diff
@@ -0,0 +1,6 @@
+import dynamic from 'next/dynamic';
+
+// Don't render the editor on the server because some parts of it depend on the browser API.
+const DynamicTextEditor = dynamic(() => import('./TextEditor'), { ssr: false });
```

Comment:
We can remove dynamic loading, cuz it causes some CLS! We don't have any server components, we added is server check globally in the app

Reply:
I only dynamic load floating menus now so it won't cause CLS anymore.